docs/source/platform/operation-registry.md

```diff
@@ -386,3 +342,126 @@ By clicking on the URL listed in the `Checking for manifest changes at` message,
+
+## Migrating from `0.1-alpha.4` to `0.2.0-alpha.1`
+
+### Summary of changes
+
+The operation registry has been upgraded to improve observability and robustness. These changes include: a [**stable operation manifest location**](#manifest-storage-location), registry [**metrics in the UI**](#metrics-and-usage-statistics), apollo [**client:push diagnostics**](#operation-registration-observability), and [**variant/tag awareness**](#varianttag-awareness).
+
+These update are designed to be transparent with a smooth upgrade path. To
+facilitate the upgrade, all versions of `apollo client:push` double-write to the new
+and old storage locations and the registry plugin reads from the new location
+and uses the old location as a fallback.
+
+> Note: Upgrading the apollo cli requires some extra caution if a variant/tag is defined in the `apollo.config.js` in order to ensure the proper variant/tag during push
+
+### Upgrade path
+
+While this upgrade should not require additional work, we recommend running
+the new plugin with `dryRun` and `debug` enabled. To report metrics and use
+the new storage location, upgrade:
+
+- `apollo-server` or `apollo-server-<variant>` to `2.6.3`
+- `apollo-server-plugin-operation-registry` to `0.2.0-alpha.1`.
+
+If operations have not been registered since June 6th, the plugin will
+fallback on the old location. With `debug` enabled, the plugin will log which
+manifest is used and the operations added.
+
+#### To target variants/tags other than "current"(default, no tag)
+
+Operations are now registered against a variant/tag and retrieved from the
+manifest specific to a variant/tag. When a variant/tag is set, the operation
+will be validated against the latest schema published under that variant/tag
+and placed in that variant/tag's manifest
+
+To register operations under a specific variant/tag:
+
+- **client**: Upgrade the `apollo` CLI package to 2.13.0 (by default the variant/tag will be set to the value in the [apollo config](https://www.apollographql.com/docs/references/apollo-config/#option-1-use-the-apollo-schema-registry))
+- **server**: Ensure that a schema has been published to the specified graph variant/tag.
+  This can be done by running `apollo service:push --tag <TAG>`
+- **client**: Ensure that operations have been registered to the specified graph variant/tag.
+  This can be done by running `apollo client:push --tag <TAG>` or modifying the service in the `apollo.config.js`
+- **server**: Set the `schemaTag` field of `apollo-server-plugin-operation-registry` to the targeted graph variant/tag
+
+```js
+const server = new ApolloServer({
+  plugins: [
+    require("apollo-server-plugin-operation-registry")({
+      schemaTag: "prod" // highlight-line
+
+      // suggested before enforcing the safelist
+      debug: true,
+      dryRun: true,
+    })
+  ]
+});
+```
+
+> Note: in order to copy the operations from one tag to another, please contact the Apollo team through Intercom or your shared Slack channel
```

Comment:
I think you can say something more direct here, like "If you want to copy all previously registered operations to a new tag, please reach out to the Apollo team". I would remove the part of this message that describes the method, since not everyone has a shared Slack channel with us.